ex02.ex

```diff
@@ -30,7 +30,7 @@ defmodule Ex02 do
   # numbers, and second should be the difference                         #
   ########################################################################
 
-  list2a = your_anonymous_function(1, 2)
+  list2a = fn (num1, num2) -> [num1+num2, num1-num2] end
```

Comment:
No deductions, but in all the code here I'd like to see spaces around the operators.